lib/msal-common/test/client/ClientCredentialClient.spec.ts

```diff
@@ -79,6 +83,28 @@ describe("ClientCredentialClient unit tests", () => {
         expect(returnVal.includes(`${AADServerParamKeys.X_MS_LIB_CAPABILITY}=${ThrottlingConstants.X_MS_LIB_CAPABILITY_VALUE}`)).toBe(true);
     });
 
+    it("Adds tokenQueryParameters to the /token request", (done) => {
+        sinon.stub(ClientCredentialClient.prototype, <any>"executePostToTokenEndpoint").callsFake(async (url: string) => {
+            console.log(url);
+            expect(url.includes("/token?testParam=testValue")).toBe(true);
+            done();
+        });
+
+        const client = new ClientCredentialClient(config);
+        const clientCredentialRequest: CommonClientCredentialRequest = {
+            authority: TEST_CONFIG.validAuthority,
+            correlationId: TEST_CONFIG.CORRELATION_ID,
+            scopes: TEST_CONFIG.DEFAULT_GRAPH_SCOPE,
+            tokenQueryParameters: {
+                testParam: "testValue"
```

Comment:
Add a few more params please.

Reply:
Also ensure one of the params is empty: 

`testParam2 = ""` in which case you'd want to ignore it, i.e. the url should not contain it.